gradle.properties

```diff
@@ -11,7 +11,7 @@ cordaProductVersion = 5.1.0-INTEROP
 ##   a per module property in which case module versions can change independently.
 ## IMPORTANT:
 ## The interop feature branches track api revisions separately to the mainline branch.
-## API version of last merge from corda mainline: 12
+## API version of last merge from corda mainline: 16
 cordaApiRevision = 15
```

Comment:
We should increment our version to 16 (`cordaApiRevision = 15` -> `cordaApiRevision = 16`)

Reply:
Yeah, this is the only issue I can see